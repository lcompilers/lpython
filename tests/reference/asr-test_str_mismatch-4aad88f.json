{
    "basename": "asr-test_str_mismatch-4aad88f",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_str_mismatch.py",
    "infile_hash": "cb0d79659e231dea35ff7fb9a0b3d61ea4e61a97a084dd4ef5897a74",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_str_mismatch-4aad88f.stderr",
    "stderr_hash": "e64e38a8c35e63d2f542f67e1d133be318f46eac1f0e5aa1f187ca19",
    "returncode": 2
}