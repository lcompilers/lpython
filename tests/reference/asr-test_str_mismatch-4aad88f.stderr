semantic error: Type mismatch in binary operator; the types must be compatible
 --> tests/errors/test_str_mismatch.py:5:9
  |
5 |     s = "abc" + c
  |         ^^^^^   ^ type mismatch (str and c32)
